### ISSUE TITLE
Field input error is not announced in full - Hours Block

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#8926]
# Problem

Siteimprove has surfaced accessibility issues with the uiowa_hours block.

- ~Error: Page region missing an accessible name~ - Not seeing this reported anymore
Example pages: https://recserv.uiowa.edu/climbing-wall
- The error is for “Field input error is not announced in full”
  - “Field inputs that are marked as aria-live="assertive" cause a screen reader to interrupt the user when an error occurs. Marking a field input as aria-atomic means that the screen reader announces the complete error message each time it occurs.”
  - https://recserv.uiowa.edu/hours
    - https://my2.siteimprove.com/Inspector/1001224/A11Y/Page?pageId=47298859852&impmd=264ED37B65FA6186129334A508391970&issue=cantTell+failed&wcag=twopointtwo#/sia-r54/failed/bVpGTVVnSng=;MmRlZmE5YmY=
  - https://recserv.uiowa.edu/intramural-sports
    - https://my2.siteimprove.com/Inspector/1001224/A11Y/Page?pageId=47299246744&impmd=264ED37B65FA6186129334A508391970&issue=failed&decision=dismissed+falsePositive+none&comp=ada#/sia-r54/failed/YXhaZnlJb0Y=;NTExZmJkNDM=
  - https://recserv.uiowa.edu/outdoor-rental-center
  - https://recserv.uiowa.edu/aquatics

Contact @bewoldt when these are fixed.

**Comment by joewhitsitt** [comment]
Moving forward with adding aria-atomic true via #1899 should resolve part of this issue.

>Page region missing an accessible name

I am seeing issues with non-unique inputs and submit buttons as well when multiple are placed on a page.
